packages/components/src/toolbar/style.scss

```diff
@@ -62,7 +62,7 @@
 				background: transparent;
 			}
 
-			&::before {
+			&:not(:active)::before {
```

Comment:
@jasmussen Is this the right way? I'm not sure :)

Reply:
I can't immediately grok what it does. But my instinct was to simply add a separate :active modifier and make the text white there. But I can see that the following CSS would interfere with that:

```
.components-button:not([aria-disabled="true"]):active {
    color: inherit;
}
```

So if what you have here works, I think it's good to ship. I don't know if it needs a comment.
Nice work.